src/coreclr/jit/importer.cpp

```diff
@@ -4298,12 +4300,24 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                         CORINFO_CLASS_HANDLE hClass = gtGetHelperArgClassHandle(call->gtCallArgs->GetNode());
                         if (hClass != NO_CLASS_HANDLE)
                         {
-                            retNode =
-                                gtNewIconNode((eeIsValueClass(hClass) &&
-                                               // pointers are not value types (e.g. typeof(int*).IsValueType is false)
-                                               info.compCompHnd->asCorInfoType(hClass) != CORINFO_TYPE_PTR)
-                                                  ? 1
-                                                  : 0);
+                            switch (ni)
+                            {
+                                case NI_System_Type_get_IsValueType:
```

Comment:
additional switch and case scopes leave even less room for clang-format so it ends up with hard-to-read formatting 🙁